grunt/connect.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
+
+var rewrite = require( "connect-modrewrite" );
+var serveStatic = require( "serve-static" );
+var rewriteMiddleware = rewrite( [
+    "^[^\\.]*$ /index.html [L]"
```

Comment:
Any request that's not specifically for a file (has a dot in the url), returns the index file